axisregistry/softness.textproto

```diff
@@ -14,7 +14,7 @@ fallback {
   value: 50
 }
 fallback {
-  name: "Extra Soft"
+  name: "SuperSoft"
```

Comment:
Shouldn't it be "Super Soft". 

What's wrong with "Extra". We use this in both the weight and width particles.

Reply:
See https://github.com/google/fonts/issues/2760 for an explanation.

At least on Mac, weights are sorted correctly, probably using OS/2 data. But, these SOFT style sub-families are part of family names, so they won't/can't sort properly without alphabetic names.

Reply:
@m4rc1e I think its better to not have spaces in fallback names; I'd like to remove all of them, as we had in static fonts' style names, because then it keeps it easy to identify each particle as a unit.

Reply:
@davelab6 When we load the axisreg in fontbakery and gftools, we [remove the spaces in the particle names](https://github.com/googlefonts/fontbakery/blob/master/Lib/fontbakery/profiles/googlefonts_conditions.py#L561-L562). I don't want a situation where we're sending PRs with a naming schema which isn't in line with what already exists. I have no problems if we go and remove all spaces in the existing particles then merge this.